app/controllers/users_controller.rb

```diff
@@ -1,8 +1,7 @@
 class UsersController < ApplicationController
-  before_action :check_is_logged_in, :check_is_admin_permission,
-    only: [:index, :show, :new, :create, :edit, :update, :destroy]
-  before_action :check_is_logged_in, only: [:edit_profile]
-  before_action :get_user, only: [ :show, :edit, :update, :destroy]
+  before_action :check_is_admin_permission, expect: %i[edit_profile]
```

Comment:
typo: `except` not `expect` [reference](https://apidock.com/rails/ActionController/Filters/ClassMethods/before_filter)